src/compiler/bundle/app-data-plugin.ts

```diff
@@ -60,6 +62,20 @@ export const appDataPlugin = (
         appendEnv(config, s);
         return s.toString();
       }
+      if (id === config.globalScript) {
```

Comment:
👋 Quick thought: you could flatten this out and return early, so:
```
if (id !== config.globalScript) {
  return null;
} 

const mod = compilterCtx.moduleMap.get(......
```

Reply:
Thanks! Yep - less indentation is a winner :)

Reply:
Cool! Thanks for this PR - hopefully it'll get accepted - I'll check it out later and test it here if I get a chance